protocol/0035-LIQM-liquidity_monitoring.md

```diff
@@ -40,21 +39,17 @@ If an incoming order would match orders on the book resulting in trades increasi
 
 ### Decreasing supplied stake
 
-If the [liquidity provision transaction would decrease](./0044-LIME-lp_mechanics.md#liquidity-provider-proposes-to-amend-commitment-amount) `supplied_stake` so that liquidity auction gets triggered then the liquidity provision amendment should be rejected and market should stay in it's current trading mode.
+If the [liquidity provision transaction would decrease](./0044-LIME-lp_mechanics.md#liquidity-provider-proposes-to-amend-commitment-amount) `supplied_stake` so that liquidity auction gets triggered then a liquidity auction is triggered the next time the next time conditions for liquidity auctions are evaluated. 
```

Comment:
No more lock-in at all then? I would worry the combination of no lock in and still having liquidity auctions might mean in a crashing market the LPs all pull out and leave people with positions in the market trapped in a liquidity auction?

Reply:
Might it be possible for liquidity auctions to periodically allow uncrossing of some trades which reduce market OI without actually exiting the auction (I.e it reduces but just not enough to drop below target)

Reply:
That might breach the simplicity rules though..

Reply:
> No more lock-in at all then? I would worry the combination of no lock in and still having liquidity auctions might mean in a crashing market the LPs all pull out and leave people with positions in the market trapped in a liquidity auction?

No lock in but you do lose some or all of your bond for quitting if below target stake.

Reply:
> Might it be possible for liquidity auctions to periodically allow uncrossing of some trades which reduce market OI without actually exiting the auction (I.e it reduces but just not enough to drop below target)

I like the idea of an uncrossing where the market remains in auction after uncrossing, that can only occur if OI is reduced. @TomMcL would you be able to take a look at how this would fit with the relevant liquidity monitoring and auction specs?